modules/apps/client-extension/client-extension-web/src/main/resources/META-INF/resources/admin/view_theme_css.jsp

```diff
@@ -16,7 +16,12 @@ for (Method method : methods) {
 	CETProperty cetProperty = method.getAnnotation(CETProperty.class);
 	String label = viewClientExtensionEntryDisplayContext.getLabel(method);
 	String name = cetProperty.name();
+	String type = viewClientExtensionEntryDisplayContext.getType();
 	Object value = viewClientExtensionEntryDisplayContext.getValue(method);
+
+	if (!FeatureFlagManagerUtil.isEnabled("LPD-34650") && type.equals(ClientExtensionEntryConstants.TYPE_THEME_CSS) && name.equals("scope")) {
+		continue;
+	}
```

Comment:
`*DisplayContext` classes seems to be responsible for the communication between `*.jsp` and the others downstream services. What do you think about move this logic to `ViewClientExtensionEntryDisplayContext`? :thinking:

Reply:
@algobob  I did not put it there because that context generic. It handles the view of any type of Frontend CX. Since this is going to be removed once the feature flag is removed, it's fine.

Is it okay for you?